### PR TITLE
Migrate Message step of Send to the new design - Closes #762

### LINKED ISSUE #762: Migrate Message step of Send to the new design
This is a subtask of #746
The goal is to migrate Message step of Send to the new design according to the designs here:
https://projects.invisionapp.com/d/main#/console/17093670/354991191/preview

### PR DESCRIPTION
**Pull request by colado**
# What was the bug or feature?
Described in #762 

### How did I fix it?
Added UI changes.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Navigate to Send tab and follow the steps until Message screen.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes

### CODE REVIEW

File: locales/resources/de.json [hunk 1]
```diff
@@ -188,4 +188,4 @@
   "Unparsed Address": "Unbekannte Adresse",
   "Choose an authenticated method": "",
   "Info": "Info"
-}
+}
```
Comment (reyraa) [inline locales/resources/de.json]: We follow a standard to add an empty line at the end of file. I don't know why eslint didn't complain here. please add the empty line here and in the other file, and I'll make sure eslint works fine in the future. Thanks

Comment (colado) [reply locales/resources/de.json]: I added an empty line to both json files now.

**Comment by colado** [comment]
@reyraa The circular progress bar is now integrated. Let me know if anything needs to be changed.

**Comment by colado** [comment]
@reyraa all commits are verified now. Feel free to merge.

**Pull request merged**